build/build.sh

```diff
@@ -32,6 +32,9 @@ if [ "$FULL_BUILD" == "true" ]; then
   python "$BUILD_DIR/set_build.py"
 fi
 
+pip install git+https://github.com/lbryio/lbryschema.git
```

Comment:
it's because when `pip install git+...`'ing, pip only looks at setup.py, so if the thing you're installing also depends on a non-pypy package it needs to be installed first.